uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/admin-dashboard/modules/settings.js

```diff
@@ -5,6 +5,7 @@
     'common/angular'
   ], function (angular) {
     return angular.module('admin-dashboard.settings', []).constant('settings', {
+      contactId: CRM.vars.leaveAndAbsences.loggedInUserId,
```

Comment:
Should we refactor and change `contactId` to `loggedInUserId` instead (as well as in component)? This gets really confusing as `contactId` should mean the currently being browsed contact (by either admin or manager) as it is implemented in most places.